### ISSUE TITLE
Support key != value  query option on API : Rawdata

### ISSUE THREAD

**Issue by itskshitiz321** [hotosm/galaxy-api#191]
Explanation : 
Export tool has functionality for supporting key != value , type of query as well , Such type of request are rare , but yet will be needed to exactly match the functionalities , for example this will be useful for such request which say , let's say I want all amenities features without toilet . Implementation will require discussion how we can add this in current API request body , could be of low priority : flagging this so that we will not miss this

**Comment by ramyaragupathy** [comment]
Looks like such negated queries are supported only through YAML specification? Could you please confirm @itskshitiz321 ?

**Comment by itskshitiz321** [comment]
yes true @ramyaragupathy  ! It is  not defined on predefined set of list
